src/query/catalog/src/table_context.rs

```diff
@@ -54,6 +54,12 @@ pub struct ProcessInfo {
     pub created_time: SystemTime,
 }
 
+#[derive(Debug, Clone)]
+pub struct SideloadOptions {
```

Comment:
can `uri` and `stage` be unified as a uri field? if it's started as `@`, it's a staged file's path

Reply:
Changed to stage only.